apps/api/src/server/appeals/appeals/appeals.formatter.js

```diff
@@ -82,7 +87,20 @@ const appealFormatter = {
 			siteVisit: {
 				visitDate: appeal.siteVisit?.visitDate || null
 			},
-			startedAt: appeal.startedAt
+			startedAt: appeal.startedAt,
+			documentationSummary: {
+				appellantCase: {
+					status: appeal.appealDetailsFromAppellant
+						? DOCUMENT_STATUS_RECEIVED
+						: DOCUMENT_STATUS_NOT_RECEIVED
+				},
+				lpaQuestionnaire: {
+					status: appeal.lpaQuestionnaire ? DOCUMENT_STATUS_RECEIVED : DOCUMENT_STATUS_NOT_RECEIVED,
+					...(appeal.appealTimetable?.questionnaireDueDate && {
+						dueDate: appeal.appealTimetable?.questionnaireDueDate
```

Comment:
It would probably be clearer for `dueDate` to always be included in the response and set to `null` if it's not got a value. 

That way we'd return a more consistent response from the API.

Reply:
done!